### PR TITLE
Ian: The Next Generation

### PR DESCRIPTION
**Pull request by Cerebulon**
New Ian sprites, again.
The ancient ones were replaced a couple of years ago but I think these from Aurora are more actually corgi-like. Hey, in-universe it's been like 9 years. This is his son, Ian Jr. Jr.

Also some improved cat inhands/onheads and dog types from Aurora. And Penny/Crusher/Columbo but not actually implemented anywhere but usable because they're neat. No we are not adding more department pets.

### CODE REVIEW

File: code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm [hunk 1]
```diff
@@ -244,5 +242,27 @@
 	icon_living = "brittany"
 	icon_dead = "brittany_dead"
 
+/mob/living/simple_mob/animal/passive/dog/german
+	name = "german shepherd"
+	real_name = "german shepherd"
+	desc = "It's a german sherpherd."
```
Comment (Sypsoti) [inline code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm:248]: What's a german *sherpherd*?

File: code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm [hunk 2]
```diff
@@ -244,5 +242,27 @@
...
+	icon_state = "german"
+	icon_living = "german"
+	icon_dead = "german_dead"
+
+/mob/living/simple_mob/animal/passive/dog/german/columbo
+	desc = "It's a german sherpherd. This one is wearing a security vest."
```
Comment (Sypsoti) [inline code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm:254]: Sherpherd.

**Review by Sypsoti — COMMENTED** [review]
Sprites are fine, typos are not.

File: code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm (hunk 1 above)
Comment (Atermonera) [reply code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm:248]: Obviously a misspelling of `german shampoodle`

**Review by Spookerton — APPROVED** [review]

File: code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm (hunk 2 above)
Comment (Spookerton) [reply code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm]: resolved

File: code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm (hunk 1 above)
Comment (Spookerton) [reply code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm]: resolved

**Comment by Spookerton** [comment]
tbh I feel the corgi at least is an objectively worse sprite, but shrug

**Review by Spookerton — CHANGES_REQUESTED** [review]
newly has an icon conflict to resolve, sorry bb

**Comment by Spookerton** [comment]
oh dear
never do that

**Review by Spookerton — APPROVED** [review]

**Review by Spookerton — CHANGES_REQUESTED** [review]
needs conflict resolution; suggest waiting on #8743 and #8740 since this will conflict with those.

**Comment by Cerebulon** [comment]
Ok, have replaced the adult corgi sprites with edited versions of https://github.com/Aurorastation/Aurora.3/pull/14870 which is generally agreed to be more gooder. Basically just added bowtie Ian/Lisa versions consistent with the current Aurora ones originally being ported, and added movement states.

**Review by Spookerton — APPROVED** [review]

**Pull request merged**